website/docs/r/domain.html.markdown

```diff
@@ -24,7 +24,11 @@ The following arguments are supported:
 
 - `name` - (Required) Custom Domain name.
 
-- `verify` - (Optional) Indicates whether the domain should be verified. 
+- `certificate_source_type` - (Optional) Certificate source type that indicates whether the certificate is provided by the user or Okta. Accepted values: `MANUAL`, `OKTA_MANAGED`. Default value = `MANUAL`
+
+  > Warning: Use of `OKTA_MANAGED` requires a feature flag to be enabled.
```

Comment:
~> **WARNING:** Use of `OKTA_MANAGED` requires a feature flag to be enabled.

Reply:
Try a refresh now.

Reply:
you're missing `~` at the beginning

Reply:
Interesting. Haven't seen that Markdown syntax before.  Updated

Originally should be similar to:
![image](https://user-images.githubusercontent.com/739719/149188691-7dc614c1-e64a-452c-9dbc-4c698f3852a9.png)

New version looks like:
![image](https://user-images.githubusercontent.com/739719/149188650-106c6680-696f-43da-b340-bc4b78693d50.png)

Reply:
Yeah, that's for Hashicorp's registry website

Reply:
That makes sense. Good to know. Thanks!